### PR TITLE
Add support for z/OS platforms to Jenkins pipelines

### PR DESCRIPTION
**Pull request by vsebe**
- enable pipelines for z/OS platforms
- reformat variables file: add support for OpenJDK repositories per spec

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
@jdekonin please review

File: buildenv/jenkins/common/build [hunk 1]
```diff
@@ -43,7 +43,9 @@ def get_source() {
             OMR_BRANCH_OPTION = (OMR_BRANCH != "")? "-omr-branch=${OMR_BRANCH}" : ""
             OMR_SHA_OPTION = (OMR_SHA != "") ? "-omr-sha=${OMR_SHA}" : ""
 
-            if (USER_CREDENTIALS_ID != '') {
+            // use sshagent with Jenkins credentials ID for all platforms except zOS
+            // on zOS use the user's ssh key
```
[the remark this answers is not in the document]
Comment (vsebe) [reply buildenv/jenkins/common/build]: Yes (prompts for passphrase). Yes, we need the rsa public key. Nope.

**Comment by vsebe** [comment]
> Does the default OPENJDK_REPO get used for all specs unless OPENJDK_REPO_SPEC exists?

Correct!

**Comment by vsebe** [comment]
@pshipton please review, thank you!

**Comment by pshipton** [comment]
jenkins compile plinux jdk8,jdk11

**Comment by pshipton** [comment]
jenkins compile plinux jdk8

**Comment by pshipton** [comment]
jenkins compile xlinux jdk8

**Comment by AdamBrousseau** [comment]
jenkins compile plinux jdk11

**Comment by AdamBrousseau** [comment]
jenkins compile plinux jdk11

**Comment by pshipton** [comment]
The PR testing appears passed, although this PR didn't get updated with the status. However I'm worried about merging it and breaking something while we are trying to finalize the 0.11.0 release.

**Pull request merged**